### PR TITLE
complete task, count characters

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * The goal is test the count characters methods.
+ *
+ * @author Carlos Orellana.
+ * @since 07/06/2018.
+ */
+public class KataCountCharactersTest {
```
Comment (carledriss) [inline src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java]: Your implementation is failing with the following unit test
```java
        int numberCharacters = this.kataCountCharacters.countCharacteres("fizzbuzz", 'Z');
        Assert.assertEquals(4, numberCharacters);
```

File: src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    private KataCountCharacters kataCountCharacters;
+    private static final int NUMBER_TWO = 2;
+    private static final int NUMBER_THIRTY_THREE = 33;
```
Comment (carledriss) [inline src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java]: You should declare this constants as local final variables inside method.

File: src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        this.kataCountCharacters = new KataCountCharacters();
+    }
+
+    /**
+     * Runs test case countCharacteresElephantSentence.
+     */
+    @Test
+    public void countCharacteresElephantSentence() {
+        int numberCharacters = this.kataCountCharacters.countCharacteres("elephant", 'e');
+        Assert.assertEquals(NUMBER_TWO, numberCharacters);
```
Comment (carledriss) [inline src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java]: Consider using a static import for Assert

File: src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java (hunk 1 above)
Comment (SDFC-Carlos-Orellana) [reply src/test/java/org/minions/devfund/carlosorellana/KataCountCharactersTest.java]: I modify my code to works correctly

**Review by carledriss — APPROVED** [review]
LGTM! 👍

**Pull request merged**